### PR TITLE
OCPBUGS#10296: Change the IPsec related text

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-10296

Link to docs preview: https://57376--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html

https://57376--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-encryption_configuring-ipsec-ovn

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@anuragthehatter Could you PTAL at this PR?

**Comment by anuragthehatter** [comment]
looks good to me. Not sure how we escaped this critical bug into docs :) Thanks for correcting it.

/lgtm

File: modules/nw-ovn-ipsec-encryption.adoc [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 :_content-type: CONCEPT
 [id="nw-ovn-ipsec-encryption_{context}"]
-= Encryption protocol and tunnel mode for IPsec
+= Encryption protocol and IPsec mode
 
 The encrypt cipher used is `AES-GCM-16-256`. The integrity check value (ICV) is `16` bytes. The key length is `256` bits.
 
-The IPsec tunnel mode used is _Transport mode_, a mode that encrypts end-to-end communication.
+The IPsec mode used is _Transport mode_, a mode that encrypts end-to-end communication by adding an Encapsulated Security Payload (ESP) header to the IP header of original packet and encrypts the packet data. {product-title} does not currently use and support IPsec _Tunnel mode_ for pod-to-pod communication.
```
Comment (eohartman) [inline modules/nw-ovn-ipsec-encryption.adoc:11]: Change "and" to "or" in "...use and support IPsec..."

Comment (eohartman) [inline modules/nw-ovn-ipsec-encryption.adoc:11]: Add a "the" before "original packet."

Thank you!

**Review by eohartman — CHANGES_REQUESTED** [review]

File: modules/nw-ovn-ipsec-encryption.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/nw-ovn-ipsec-encryption.adoc]: Addressed

Comment (xenolinux) [reply modules/nw-ovn-ipsec-encryption.adoc]: Addressed

**Comment by snarayan-redhat** [comment]
@xenolinux Please create manual CPs for 4.11, 4.10 and 4.9

**Pull request merged**